crates/ide_assists/src/handlers/remove_dbg.rs

```diff
@@ -415,6 +454,50 @@ fn main() {
 }"#,
             r#"fn foo() {
     match &x {}
+}"#,
+        );
+    }
+
+    #[test]
+    fn test_remove_empty_dbg() {
+        check_assist(remove_dbg, r#"fn foo() { $0dbg!(); }"#, r#"fn foo() { }"#);
+        check_assist(
+            remove_dbg,
+            r#"fn foo() {
+$0dbg!();
+}"#,
+            r#"fn foo() {
+}"#,
+        );
```

Comment:
Let's use standard test style

```suggestion
            r#"
fn foo() {
    $0dbg!();
}
"#,
            r#"
fn foo() {
}
"#,
        );
```

Although, maybe just write this as a single line.

Reply:
I guess multiline test shows better, than newline before dbg is removed